docs/best-practices.md

```diff
@@ -0,0 +1,51 @@
+# Best practices for writing actors
+
+
+### Follow the Python Coding Guidelines
+
+See the [Python Coding Guidelines](CONTRIBUTING.md#coding-guidelines).
+
+### Avoid running the code on a module level
```

Comment:
Mention that some code can be executed on a module level, for example constant definition. Such code should be simple, fast and with no side effects though.